githubteacher-fuquay.md

```diff
@@ -0,0 +1,5 @@
+## Fuquay Varina, NC
+
+### Places to eat
```

Comment:
Not everyone can use gitter.  I can't.  I can't access github.com for the clone and unable to tell anyone because I can't communicate